Hazel/Dtls/DtlsConnectionListener.cs

```diff
@@ -1285,7 +1287,7 @@ public override void DisconnectOldConnections(TimeSpan maxAge, MessageWriter dis
             foreach (KeyValuePair<IPEndPoint, PeerData> kvp in this.existingPeers)
             {
                 PeerData peer = kvp.Value;
-                lock(peer)
+                lock (peer)
```

Comment:
I thought about putting the peer remove here as a sweep, but this is very much meant for one dumb task. And sweeping it would be dumb anyway.

Reply:
Yeah I think that's a good call--not sure how this library works with a temporary disconnection from the same source IP, but as the existingPeers dictionary seems to be keyed by IP it makes sense to me to clean that up as soon as we know the connection is dropped